isis/src/galileo/apps/gllssi2isis/main.cpp

```diff
@@ -110,22 +109,15 @@ void IsisMain() {
     p.StartProcess();
   }
   else {
-    summedOutput = new Cube();
-    summedOutput->setDimensions(p.Samples() / 2, p.Lines() / 2, p.Bands());
-    summedOutput->setPixelType(p.PixelType());
-    summedOutput->create(ui.GetFileName("TO"));
-    p.StartProcess(translateData);
+    p.SetDimensions(p.Samples() / 2, p.Lines() / 2, p.Bands());
+    summedOutput = p.SetOutputCube("TO");
```

Comment:
ocube = p.SetOutputCube("TO"); ?

Do we need summedOutput now...?

Reply:
I just noticed that summedOutput is being used further down. Is that why we still need to instantiate it here?